sdk/cognitiveservices/azopenai/models.go

```diff
@@ -320,11 +420,11 @@ type Embeddings struct {
 // EmbeddingsOptions - The configuration information for an embeddings request. Embeddings measure the relatedness of text
 // strings and are commonly used for search, clustering, recommendations, and other similar scenarios.
 type EmbeddingsOptions struct {
-	// REQUIRED; Input text to get embeddings for, encoded as a string. To get embeddings for multiple inputs in a single request,
-	// pass an array of strings. Each input must not exceed 2048 tokens in length.
+	// REQUIRED; Input texts to get embeddings for, encoded as a an array of strings. Each input must not exceed 2048 tokens in
+	// length.
 	// Unless you are embedding code, we suggest replacing newlines (\n) in your input with a single space, as we have observed
 	// inferior results when newlines are present.
-	Input any
+	Input []*string
```

Comment:
I believe we now want slices of scalar types to not be pointer-to-type.

Reply:
That will likely have to wait for a code generator update to opt into the behavior though.

Reply:
https://github.com/Azure/autorest.go/pull/986

Reply:
Issue filed: https://github.com/Azure/azure-sdk-for-go/issues/21101